app/util/common/config/platform.go

```diff
@@ -120,100 +110,39 @@ func (p *platformConfig) Dump() (map[string]interface{}, error) {
 }
 
 func (p *platformConfig) GitRemoteName() (string, error) {
-	return p.GetValue(gitRemoteName)
-}
-
-func (p *platformConfig) GitServerHost() (string, error) {
-	return p.GetValue(gitServerHost)
-}
-
-func (p *platformConfig) GitServerPort() (string, error) {
-	return p.GetValue(gitServerPort)
-}
-
-func (p *platformConfig) GitServerProtocol() (string, error) {
-	return p.GetValue(gitServerProtocol)
+	return p.GetValue(gitServerRemote)
 }
 
 func (p *platformConfig) GitServerAddr() (string, error) {
-
-	protocol, err := p.GitServerProtocol()
-	if err != nil {
-		return "", errors.Trace(err)
-	}
-
-	host, err := p.GitServerHost()
-	if err != nil {
-		return "", errors.Trace(err)
-	}
-
-	addr := protocol + "://" + host
-	port, err := p.GitServerPort()
-	if err != nil || port == "" {
-		return addr, nil
-	}
-	return addr + ":" + port, nil
-}
-
-func (p *platformConfig) Address() (string, error) {
-	addr, err := p.GetValue(platformServerAddr)
-	if err != nil {
-		return "", errors.Trace(err)
-	}
-
-	port, err := p.GetValue(platformServerPort)
+	addr, err := p.GetValue(gitServerAddr)
 	if err != nil {
 		return "", errors.Trace(err)
 	}
-
-	return addr + ":" + port, nil
+	return addr, nil
```

Comment:
We should probably just return p.GetValue() directly. But that begs the question, do we need these functions at all? Why not just call p.GetValue(gitServerAddr) instead of p.Address(), given gitServerAddr is a global and will be available.